### ISSUE TITLE
PDF generator fails running in enviroments without X

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#1434]
Trying to generate a PDF in a *nix without a X server , the javaVM  crashes with the following message:

X connection to localhost:10.0 broken (explicit kill or server shutdown).
JVM exited unexpectedly.

To solve it, it's needed to add the next line to the wrapper.conf:

wrapper.java.additional.1=-Djava.awt.headless=true

Imported Issue. Original Details:
Jira Issue Key: MIRTH-1435
Reporter: albertosaez
Created: 2010-06-02T00:39:14.000-0700

**Comment by rbeckman-nextgen** [comment]
Our appliances are running Linux without X server installed, and we have had customers using channels that generate PDFs without running into this issue. Which version of the JVM and which Linux distribution are you using?

Imported Comment. Original Details:
Author: geraldb
Created: 2010-06-03T15:10:19.000-0700

**Comment by rbeckman-nextgen** [comment]
I've investigated the issue.

If the mirth process is started with the X environment set (for example, in a remote ssh session with the "-X" option, then java uses the X for some awt ops (unless you set the headless mode)

In my case, I started mirth with a X-tunneled ssh session and then I log out. When I ran the mirth server, it crashed because the X server was not ready by then.

Imported Comment. Original Details:
Author: albertosaez
Created: 2010-06-03T15:27:04.000-0700
